source/dub/internal/vibecompat/inet/path.d

```diff
@@ -7,9 +7,6 @@
 */
 module dub.internal.vibecompat.inet.path;
 
-version (Have_vibe_core) public import vibe.core.path;
```

Comment:
unrelated removal?

Reply:
No it's related, but as you can see in a separate commit. Otherwise we have inconsistent attributes.